package com.esferalia.aon.gwt.payroll.client;

import com.esferalia.aon.gwt.common.client.widget.CustomDialog;
import com.google.gwt.core.shared.GWT;
import com.google.gwt.resources.client.CssResource;
import com.google.gwt.uibinder.client.UiBinder;
import com.google.gwt.uibinder.client.UiField;
import com.google.gwt.user.client.ui.Widget;

public class ContractJourneyDialog extends CustomDialog {

	interface Binder extends UiBinder<Widget, ContractJourneyDialog> {

	}

	private static final Binder binder = GWT.create(Binder.class);
	
	@UiField
	MyStyle style;

	interface MyStyle extends CssResource {
		
	}

	public ContractJourneyDialog() {
		setCaption("DURACION DE LA JORNADA");
		
		setWidget(binder.createAndBindUi(this));		
	}

}
